GoogleTestAdapter/Core/TestCases/TestCaseFactory.cs

```diff
@@ -234,6 +234,7 @@ private TestCase CreateTestCase(TestCaseDescriptor descriptor)
             var testCase = new TestCase(
                 descriptor.FullyQualifiedName, descriptor.FullyQualifiedName, _executable, descriptor.DisplayName, "", 0);
             testCase.Traits.AddRange(GetFinalTraits(descriptor.DisplayName, new List<Trait>()));
+            testCase.Traits.Add(new Trait(nameof(TestCaseDescriptor.TestType), descriptor.TestType.ToString()));
```

Comment:
Might be worth adding a small comment somewhere explaining why we add the test type to the traits in the first place.

Reply:
Changes made. Please re-approve.